Topic 1/src/point7/Music.java

```diff
@@ -0,0 +1,78 @@
+package point7;
+
+public class Music {
+	private String disk; // indicates which disk or album the song belongs
+	private String artist;
+	private String title;
+	private int duration;
+	private boolean state; // indicates whether this on or off
```

Comment:
[High] It is possible to have more than 2 states.
For example,
 - the user can start listening a song if there is no music already being played or it is in *pause*
 - the user can pause a song (only if there is the player is playing music)
 - the user can stop a song if there is a song being played.